go.mod

```diff
@@ -3,12 +3,7 @@ module github.com/unidoc/unihtml-src/examples
 go 1.15
```

Comment:
If I'm not mistaken, I've updated this version to 1.17 in `unihtml-src`, could you update this go.mod and go.sum file?